platform/linux-generic/pktio/netmap.c

```diff
@@ -526,6 +526,10 @@ static int netmap_start(pktio_entry_t *pktio_entry)
 
 	base_desc.self = &base_desc;
 	base_desc.mem = NULL;
+	if (pktio_entry->s.pkt_nm.is_virtual) {
+		base_desc.req.nr_rx_slots = 256;
+		base_desc.req.nr_tx_slots = 64;
```

Comment:
This is good to be placed to odp config.

Reply:
Thanks for the review! Can you explain a bit more on how to find information about ODP config and how to use it? I guess making this configurable could be a good addition. It probably won't make it to Tiger Moth LTS, though.

Reply:
The runtime configuration support was added in commit 534667daec58ce8b2e417709e788934bea6f8f09. You can take a look at e.g. dpdk pktio as it uses libconfig to configure the number of rx/tx descriptors.

According to netmap_user.h, NM_OPEN_RING_CFG flag should be used when passing these options, so I'm wondering if this is really working.

Reply:
This could be a common option for all netmap device types and not only for virtual devices. In the configuration file the default value should probably be 0 -> use default values defined by netmap.

Reply:
My understanding is that these affect the operation of virtual devices only, i.e. that the is_virtual check is there only for safety reasons (I didn't have 40GbE NICs at my disposal when writing the patch and didn't want to take the risk of reducing their performance even though I was 99% sure it doesn't affect them).

But thanks for the information! When I have time, I will take a look at the configuration support. The performance results are repeatable, and I could observe genuine performance increase, so I suspect they are working.